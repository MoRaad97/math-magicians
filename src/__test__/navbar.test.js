import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../Components/navbar';

it('render the Calculator Component', () => {
  const tree = renderer.create(<BrowserRouter><Navbar /></BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
