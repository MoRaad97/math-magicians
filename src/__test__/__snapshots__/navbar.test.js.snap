// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the Calculator Component 1`] = `
<nav>
  <div
    className="nav-title"
  >
    <h1>
      Math-Magicians
    </h1>
  </div>
  <div
    className="links"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </div>
</nav>
`;
